.vib/kong/goss/vars.yaml

```diff
@@ -8,10 +8,6 @@ directories:
       - /opt/bitnami/kong/conf
   - paths:
       - /docker-entrypoint-initdb.d
-      - /opt/bitnami/kong/openresty/luajit/share/lua/5.1/kong/include/opentelemetry
+      - /opt/bitnami/kong/include/opentelemetry
```

Comment:
We are already checking this path is a symlink in `kong.yaml`, I don't think we need to check this here.